src/components/common/CitiesList/AllCitiesList.jsx

```diff
@@ -0,0 +1,13 @@
+import useAllCities from '../../../hooks/useAllCities';
+import CitiesList from './CitiesList';
+
+const AllCitiesList = ({ limit, increaseLimit, showLoadMoreButton }) => (
+  <CitiesList
+    limit={limit}
+    increaseLimit={increaseLimit}
+    showLoadMoreButton={showLoadMoreButton}
+    useCitiesHook={useAllCities}
```

Comment:
You should not pass a hook as prop

https://stackoverflow.com/questions/66800263/passing-a-hook-as-a-prop